src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -99,9 +99,31 @@ public void ChatTestsStaff_003_sendPrivateMessage() {
 
   @DontRun(env = {"prod"})
   @Test(groups = {"ChatTestsStaff_004", "ChatStaff", "ChatTests"})
-  public void ChatTestsStaff_004_basicUserChatFails() {
+  public void basicUserChatFailsOnPreviewEnvirement() {
     openChatForUser(credentials.userName10, credentials.password10);
     SpecialPageObject special = new SpecialPageObject();
     special.verifyPageHeader("Permissions error");
   }
+
+  @DontRun(env = {"sandbox"})
+  @Test(groups = {"ChatTestsStaff_005", "ChatStaff", "ChatTests"})
+  public void staffOptionsNotDisplayed() {
```

Comment:
there are three types of displaying options
a) user to user or staff
b) staf to user
c) staf to staff 
All that casts needs to be covered by tests by verifications or just functionality so I think this test is needed because other ways are tested by options functionality, this test is only verification test to do not double functionality tests I will just change the test name to staffOptionsAreNotDisplayedOnOtherStaffUser